code/modules/vehicles/mecha/equipment/weapons/weapons.dm

```diff
@@ -91,12 +91,14 @@
 	harmful = TRUE
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/disabler
-	equip_cooldown = 8
+	equip_cooldown = 1.5 SECONDS
 	name = "\improper CH-DS \"Peacemaker\" disabler"
 	desc = "A weapon for combat exosuits. Shoots basic disablers."
```

Comment:
Amend this, it no longer shoots basic disablers

Reply:
Done